src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -56,6 +56,20 @@
   </div>
 
   <ng-template #cd_utilities>
+    <ng-container *ngIf="router.url === '/dashboard'">
+      <li class="nav-item">
+        <div class="form-check form-switch cd-navbar no-hover d-flex align-items-center">
+          <input class="form-check-input mb-1 blue-color"
+                 type="checkbox"
+                 [checked]="dashboardVersion === 'v3'"
+                 (change)="toggleContent()">
+          <a class="form-check-label no-hover"
+             i18n>
+            Dashboard v3
+          </a>
+        </div>
+      </li>
+    </ng-container>
```

Comment:
I'm not sure about displaying this in the top bar... It's a feature that is not meant to be changed often, so having that 100% of the time displayed there is like wasting user's attention in something that's unlikely to be used/changed.

If we take this toggle from MODULE OPTIONS (via Feature Toggles or whatever) instead, this becomes automatically displayed in the Dashboard > Manager Modules > Dashboard.

If we find that the above link is too buried into the Nav menu, then we could create a separate view for Dashboard MODULE OPTIONS.

Reply:
Went with this approach, removed the toggle button and added it to the dashboard module options